# HG changeset patch
# User andrew
# Date 1398352866 -3600
#      Thu Apr 24 16:21:06 2014 +0100
# Node ID 2b4f89de39c2c4fd09f927adacb3781454dbee44
# Parent  6cb711fa264d25f3592471a81a5b865c505a9fd7
Extend 8041658 to all files in the HotSpot build.

diff -r 6cb711fa264d -r 2b4f89de39c2 make/linux/makefiles/gcc.make
--- openjdk/hotspot/make/linux/makefiles/gcc.make	Wed Apr 23 22:06:29 2014 +0100
+++ openjdk/hotspot/make/linux/makefiles/gcc.make	Thu Apr 24 16:21:06 2014 +0100
@@ -267,7 +267,7 @@
     OPT_CFLAGS/mulnode.o += $(OPT_CFLAGS/NOOPT)
   endif
   ifeq ($(shell expr $(CC_VER_MAJOR) = 4 \& $(CC_VER_MINOR) = 9), 1)
-    OPT_CFLAGS/macroAssembler_x86.o += -fno-devirtualize
+    OPT_CFLAGS += -fno-devirtualize
   endif
 endif
 
